cypress/locators/menu-list/index.js

```diff
@@ -0,0 +1,10 @@
+import {
+  MENU_LIST_PREVIEW, MENU_TITLE, MENU_LIST_SEARCH, MENU_LIST,
+} from './locators';
+
+// component preview locators
+export const menuListPreview = () => cy.iFrame(MENU_LIST_PREVIEW);
+export const menuTitle = () => cy.iFrame(MENU_TITLE);
```

Comment:
`.first()` should be here, not in step def

Reply:
fixed